app/src/main/res/values/strings.xml

```diff
@@ -433,6 +433,7 @@
     <string name="background_player">Background player</string>
     <string name="popup_player">Popup player</string>
     <string name="always_ask_open_action">Always ask</string>
+    <string name="add_to_playlist">Add to playlist</string>
```

Comment:
We already have that string, but its name is wrong. (I think the string was changed, but its name wasn't).
Please change `append_playlist` -> `add_to_playlist`